lineman/spec-e2e/proposal_spec.coffee

```diff
@@ -61,27 +61,35 @@ describe 'Proposals', ->
       proposalsHelper.proposalActionsDropdown().click()
       proposalsHelper.proposalActionsDropdownClose().click()
       proposalsHelper.closeProposalButton().click()
-      expect(proposalsHelper.previousProposalsList().getText()).toContain('lets go hiking')
+      expect(flashHelper.flashMessage()).toContain('Proposal closed')
+      expect(proposalsHelper.previousProposalsList()).toContain('lets go hiking')
+      expect(proposalsHelper.previousProposalsList()).toContain('Closed')
+
+    it 'displays the time at which the proposal closed', ->
```

Comment:
You're not actually testing that it displays the time.. which is ok.. maybe change the test name to say "displays that the proposal is closed"